TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/common_operations.py

```diff
@@ -124,10 +124,12 @@ def get_all_index_details(cluster: Cluster, index_prefix_ignore_list=None, **kwa
     all_index_details = execute_api_call(cluster=cluster, path="/_cat/indices?format=json", **kwargs).json()
     index_dict = {}
     for index_details in all_index_details:
-        valid_index = not index_matches_ignored_index(index_name=index_details['index'],
+        index_name = index_details['index']
+        valid_index = not index_matches_ignored_index(index_name,
                                                       index_prefix_ignore_list=index_prefix_ignore_list)
         if index_prefix_ignore_list is None or valid_index:
-            index_dict[index_details['index']] = index_details
+            count_response = execute_api_call(cluster=cluster, path=f"/{index_name}/_count?format=json", **kwargs)
```

Comment:
Please add a comment to cite the following page and snip the following prose "These metrics are retrieved directly from [Lucene](https://lucene.apache.org/core/), which Elasticsearch uses internally to power indexing and search. As a result, all document counts include hidden [nested](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/nested.html) documents."

https://www.elastic.co/guide/en/elasticsearch/reference/7.10/cat-indices.html

Reply:
And THANK YOU for finally bringing to my attention WHY THESE DIFFERENCES APPEAR!  woohoo!